Fix global flake8 issues

aorenste opened the pull request:
Prior to this `lintrunner --all-files --take FLAKE8` failed.

Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #124771
* #124428



cc @mrshenli @pritamdamania87 @zhaojuanmao @satgera @rohan-varma @gqchen @aazzolini @osalpekar @jiayisuse @H-Huang @kwen2501 @awgu @penguinwu @fegin @XilunWu @wanchaol @fduwjj @wz337 @tianyu-l @wconstab @yf225 @chauhang @d4l3k @voznesenskym @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @peterbell10 @ipiszy @chenyang78 @kadeng @muchulee8 @ColinPeppler @amjames @desertfire

aorenste committed:
Fix global flake8 issues

[ghstack-poisoned]

Skylion007 reviewed (approved):
Next PR will add the file to pyproject.toml for flake8+ruff?

aorenste committed:
Update on "Fix global flake8 issues"




cc mrshenli pritamdamania87 zhaojuanmao satgera rohan-varma gqchen aazzolini osalpekar jiayisuse H-Huang kwen2501 awgu penguinwu fegin XilunWu wanchaol fduwjj wz337 tianyu-l wconstab yf225 chauhang d4l3k voznesenskym EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx peterbell10 ipiszy chenyang78 kadeng muchulee8 ColinPeppler amjames desertfire

[ghstack-poisoned]

aorenste committed:
Update on "Fix global flake8 issues"




cc mrshenli pritamdamania87 zhaojuanmao satgera rohan-varma gqchen aazzolini osalpekar jiayisuse H-Huang kwen2501 awgu penguinwu fegin XilunWu wanchaol fduwjj wz337 tianyu-l wconstab yf225 chauhang d4l3k voznesenskym EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx peterbell10 ipiszy chenyang78 kadeng muchulee8 ColinPeppler amjames desertfire

[ghstack-poisoned]

aorenste commented:
> Next PR will add the file to pyproject.toml for flake8+ruff?

I'm not sure what you mean by this...

Skylion007 commented:
> > Next PR will add the file to pyproject.toml for flake8+ruff?
> 
> I'm not sure what you mean by this...

Oh sorry, I meant make sure the files are covered by the lintrunner.toml configuration so they are checked automatically from now on by flake8 and mypy etc.

aorenste committed:
Update on "Fix global flake8 issues"

Prior to this `lintrunner --all-files --take FLAKE8` failed.




cc mrshenli pritamdamania87 zhaojuanmao satgera rohan-varma gqchen aazzolini osalpekar jiayisuse H-Huang kwen2501 awgu penguinwu fegin XilunWu wanchaol fduwjj wz337 tianyu-l wconstab yf225 chauhang d4l3k voznesenskym EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx peterbell10 ipiszy chenyang78 kadeng muchulee8 ColinPeppler amjames desertfire

[ghstack-poisoned]

aorenste commented:
@pytorchbot merge

aorenste commented:
@pytorchbot merge

aorenste committed:
Update on "Fix global flake8 issues"

Prior to this `lintrunner --all-files --take FLAKE8` failed.




cc mrshenli pritamdamania87 zhaojuanmao satgera rohan-varma gqchen aazzolini osalpekar jiayisuse H-Huang kwen2501 awgu penguinwu fegin XilunWu wanchaol fduwjj wz337 tianyu-l wconstab yf225 chauhang d4l3k voznesenskym EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx peterbell10 ipiszy chenyang78 kadeng muchulee8 ColinPeppler amjames desertfire

[ghstack-poisoned]

aorenste committed:
Update on "Fix global flake8 issues"

Prior to this `lintrunner --all-files --take FLAKE8` failed.




cc mrshenli pritamdamania87 zhaojuanmao satgera rohan-varma gqchen aazzolini osalpekar jiayisuse H-Huang kwen2501 awgu penguinwu fegin XilunWu wanchaol fduwjj wz337 tianyu-l wconstab yf225 chauhang d4l3k voznesenskym EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx peterbell10 ipiszy chenyang78 kadeng muchulee8 ColinPeppler amjames desertfire

[ghstack-poisoned]

aorenste commented:
@pytorchbot merge -f "Looks like #124496 introduced a doc error"

aorenste commented:
Relanding - was reverted because of an unrelated error in #123735

@pytorchbot merge

malfet commented on test/test_jit.py:
```diff
@@ -15679,7 +15679,7 @@ def fn(string):
         @torch.jit.script
         def test(self, a):
-            # 🤷🤷🤷🤷
+            # shrug
```
Well, this BE change actually renders test useless, as it supposed to test unicode comments... :/

aorenste replied:
Sure - It seems that we should either tell flake8 to allow unicode or we should not have code with unicode in it.
